poketechnika/MenuScene.cpp

```diff
@@ -1,19 +1,20 @@
-#include "MenuScene.h"
-#include <SFML/Graphics.hpp>
 #include <iostream>
+#include "MenuScene.h"
 
-MenuScene::MenuScene(Renderer* rend) : Scene(rend)
+MenuScene::MenuScene() 
+	: Scene{}, ih{ {}, {} }, 
+	background{ "assets/textures/background_menu.png", sf::Vector2f{0, 0} }
 {
-	std::cout << "Created SceneMenu" << std::endl;
+	std::cout << "Created menu scene" << '\n';
```

Comment:
to też nwm co za istotna zmiana, ale whatever

Reply:
To wynika z tego że przy przepisywaniu kodu najpierw usunęłam tą linijkę a w innym kolejnym commicie znów ją wpisałam odrobinę inaczej, tylko github pokazuje tutaj wszystkie commity zbiorczo i tak wyszło